src/calendar.js

```diff
@@ -316,7 +316,7 @@ class Calendar extends Component {
                   selected={cell.selected}
                   today={cell.today}
                   moment={cell.moment}
-                  onClick={::this.onNavigateCell}
+                  onClick={type !== 'header' ? ::this.onNavigateCell : () => {}}
```

Comment:
We can use lodash noop instead (Will improve readability and extra function creation per cell)